src/Catalyst.TestUtils/FailingRequestObserver.cs

```diff
@@ -49,10 +49,7 @@ public sealed class FailingRequestObserver :
             ICorrelationId correlationId)
         {
             var count = Interlocked.Increment(ref _counter);
-            if (count % 2 == 0)
-            {
-                throw new ArgumentException("something went wrong handling the request");
-            }
+            if (count % 2 == 0) throw new ArgumentException("something went wrong handling the request");
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add curly braces around the nested statement(s) in this 'if' block.](https://app.codacy.com/app/catalyst-network/Catalyst.Framework/pullRequest?prid=4612924)